modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/TracksAnalyticsTracker.kt

```diff
@@ -91,6 +92,9 @@ class TracksAnalyticsTracker @Inject constructor(
                 AppPlatform.Phone -> "phone"
                 AppPlatform.WearOs -> "watch"
             },
+            PredefinedEventProperty.APP_VERSION_NAME to settings.getVersion(),
+            PredefinedEventProperty.APP_VERSION_CODE to settings.getVersionCode(),
```

Comment:
The Tracks library should attach this data already to events. It uses `PackageInfo` for this, and `settings.getVersionCode` uses `BuildConfig` but I think the value should be the same.

https://github.com/Automattic/Automattic-Tracks-Android/blob/trunk/AutomatticTracks/src/main/java/com/automattic/android/tracks/DeviceInformation.java#L81-L82

Reply:
Ahh, thanks! I'll drop the commit. I couldn't find it in tracks previously but it is all under `device_info` section.

Reply:
Thanks! Yeah, it's confusing - it's not really a "device info" 😕